block/submit.go

```diff
@@ -43,8 +44,9 @@ func SubmitLoopInner(
 	ctx context.Context,
 	logger types.Logger,
 	bytesProduced chan int, // a channel of block and commit bytes produced
-	maxBatchSkew uint64, // max number of blocks that submitter is allowed to have pending
-	unsubmittedBlocks func() uint64,
+	maxBatchSkew time.Duration, // max number of blocks that submitter is allowed to have pending
+	unsubmittedBlocksNum func() uint64,
+	skewTime func() (time.Duration, error),
 	maxBatchTime time.Duration, // max time to allow between batches
```

Comment:
`maxBatchTime` is confusing now

```suggestion
	batchSubmitTime time.Duration, // max time to allow between batches
```